__tests__/app.test.js

```diff
@@ -183,3 +183,63 @@ describe("CORE: GET - /api/articles/:article_id", () => {
       });
     })
  
+
+
+    const input = {username : 'rogersop', body : 'big comment' }
+    const badInput = {username : 'cheeseman', body : 'big comment' }
+    const badInput2 = {username : 'rogersop', body : null }
+
```

Comment:
^^ These three should not be outside the scope of the specific test. You run the risk of accidentally using them later on. Please place them inside each test!